src/ripple/app/consensus/RCLConsensus.h

```diff
@@ -78,6 +78,17 @@ class RCLConsensus : public Consensus<RCLConsensus, RCLCxTraits>
 private:
     friend class Consensus<RCLConsensus, RCLCxTraits>;
 
+    /** Notification that a new consensus round has begun.
```

Comment:
I recommend adding to the public portion of the declaration deleted copy members:
````
RCLConsensus(RCLConsensus const&) = delete;
RCLConsensus& operator=(RCLConsensus const&) = delete;
````